### DISCUSSION TITLE
Dear author

### DISCUSSION THREAD

**Discussion by 1234f503** [Stellarium/stellarium#3935] [Ideas]
Dear author, hello. In your software, the Japanese Twenty Eight Mansions in the Landscape Culture section should actually be the Chinese Twenty Eight Mansions, because China discovered Twenty Eight Mansions earlier than Japan.

**Comment by gzotti** [comment]
As can be read in the description and in the same discussion a few weeks ago, Everybody and the dog knows that. This skyculture describes the practice in Japan.
